src/cascadia/TerminalSettings/KeyChord.cpp

```diff
@@ -4,6 +4,82 @@
 #include "pch.h"
 #include "KeyChord.h"
 
+static const std::wstring CTRL_KEY{ L"ctrl" };
+static const std::wstring SHIFT_KEY{ L"shift" };
+static const std::wstring ALT_KEY{ L"alt" };
+
+static const std::vector<std::pair<int32_t, std::wstring>> vkeyNamePairs {
```

Comment:
this looks like it could be an `unordered_map` as well

Reply:
I suggest use constexpr std::wstring_view